public_dev_meetings.md

```diff
@@ -0,0 +1,30 @@
+Gno.land Developer Call 
+
+Purpose
+
+Gno.land Developer Call is a bi-weekly meeting held by Gno core devs and dev contributors to discuss technical issues, brainstorm, and collaborate on development work with regards to the smart contract platform. 
+
+Currently, the calls are recorded and uploaded to Gno.land YouTube, however in the future they will be streamed to YouTube and/or Twitch for community members to listen to the live meeting. 
+
+
+Agenda
+
+The agenda will be posted prior to the call and the dev contributors and larger community is welcome to add topics as long as they are technical in nature, and related to the Gno.land project. 
+
+Ps: If you are new and interested in asking general questions, join our bi-weekly Gno.land Office Hours on Discord. 
+
+Attendance
+
+The current attendees are the Gno.land core development team and core contributor community. We invite active contributors interested in joining the bi-weekly meeting to reach out and let us know how they are using Gno.land or how they want to support the engineering initiatives. 
+
+Upcoming meeting calendar
```

Comment:
You can add ### to this line to make it a Header 3:
`### Upcoming meeting calendar`
```suggestion
### Upcoming meeting calendar
```